index.bs

```diff
@@ -2919,6 +2919,94 @@ partial dictionary MLOpSupportLimits {
     1. Return |output|.
 </details>
 
+
+### cumulativeSum ### {#api-mlgraphbuilder-cumulativesum}
+Compute the accumulated sum of a series of values along the given axis, either including or excluding the current value.
+
+<script type=idl>
+dictionary MLCumulativeSumOptions : MLOperatorOptions
+{
+    bool exclusive = false;
+    bool reversed = false;
+}
+partial interface MLGraphBuilder {
+  MLOperand cumulativeSum(MLOperand input,
+                          unsigned long axis,
+                          optional MLCumulativeSumOptions options = {});
+};
+
+partial dictionary MLOpSupportLimits {
+  MLSingleInputSupportLimits cumulativeSum;
+};
+</script>
+
+<table id=constraints-cumulativesum class='data' link-for="MLGraphBuilder/cumulativeSum(input, options)">
+  <caption>Constraints for {{MLGraphBuilder/cumulativeSum()}}</caption>
+  <thead>
+    <tr>
+      <th>operand</th>
+      <th>[=/allowed data types=]</th>
+      <th>[=/allowed ranks=]</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>{{input}}</td>
+    <td>[=/any data type|any=]</td>
+    <td>[=/any rank|N=]</td>
+  <tr>
+  </tr>
+  <tr>
+    <td>*output*</td>
+    <td>[=/same type as|same as=] {{input}}</td>
+    <td>[=/same rank as|same as=] {{input}}</td>
+  </tr>
+</table>
+
+{{MLCumulativeSumOptions}} has the following members:
+<dl dfn-type=dict-member dfn-for=MLCumulativeSumOptions>
+    : <dfn>exclusive</dfn>
+    ::
+        Whether to include or exclude the current value in the output, meaning inclusive presum addition (see https://en.wikipedia.org/wiki/Prefix_sum) or exclusive post-sum addition. Given input *[1,2,3,4]*, inclusive addition would yield an output of *[1,3,6,10]* whereas exclusive would yield *[0,1,3,6]*. The default is inclusive.
+
+    : <dfn>reversed</dfn>
+    ::
+        Whether to reverse the summation direction along the active axis to instead start from the high coordinate to low coordinate. Given input *[1,2,3,4]*, inclusive forward addition would yield an output of *[1,3,6,10]* whereas backward summation would yield *[10,9,7,4]*. The default is exclusive.
```

Comment:
"The default is exclusive" - should be "The default is forward." ?

Also, inconsistent phrasing "inclusive forward addition" vs. "backward summation"